deploy/terraform/outputs.tf

```diff
@@ -13,4 +13,13 @@ output "s3_state_bucket_arn" {
 output "dynamodb_state_table_name" {
   value = aws_dynamodb_table.ps_adaptors_tf_locks.name
   description = "The name of DynamoDB state table"
+}
+
+output "amq_endpoint" {
+  value = aws_mq_broker.gp2gp_ps_broker.instances.0.endpoints.1
+}
+
+output "ps_db_instance_url" {
+  value       = "jdbc:postgresql://${aws_db_instance.ps_db.endpoint}"
```

Comment:
This value is created above in `main.tf` too, would creating a [Terraform local](https://developer.hashicorp.com/terraform/language/values/locals) for it be useful?

Reply:
agree, will move it to local var